include/assimp/Bitmap.h

```diff
@@ -75,13 +75,12 @@ class ASSIMP_API Bitmap {
         uint32_t offset;
 
         // We define the struct size because sizeof(Header) might return a wrong result because of structure padding.
-        // Moreover, we must use this ugly and error prone syntax because Visual Studio neither support constexpr or sizeof(name_of_field).
-        static const std::size_t header_size =
-            sizeof(uint16_t) + // type
-            sizeof(uint32_t) + // size
-            sizeof(uint16_t) + // reserved1
-            sizeof(uint16_t) + // reserved2
-            sizeof(uint32_t);  // offset
+        static constexpr std::size_t header_size =
+            sizeof(decltype(type)) +
+            sizeof(decltype(size)) +
+            sizeof(decltype(reserved1)) +
+            sizeof(decltype(reserved2)) +
+            sizeof(decltype(offset));
```

Comment:
You shouldn't need `decltype` here; this is equivalent:

            sizeof(type) +
            sizeof(size) +
            sizeof(reserved1) +
            sizeof(reserved2) +
            sizeof(offset);

Reply:
Yep, I totally missed that somehow, thank you!